### ISSUE TITLE
DUB: Workplan

### ISSUE THREAD

**Issue by derwas** [derilinx/derilinx#37]
From ckan-docker created by [CarlQLange](https://github.com/CarlQLange) : derilinx/ckan-docker#137  

# Release blockers
- [x]  bring UI of site closer to wordpress - "seamless"
- [x]  sort by popularity, view stats for publishers
- [x]  what CKAN branch are we going ahead with
   I guess whatever master version we're on now is fine, and then move to 2.8 as soon as humanly possible.
- [x]  what release strategy are we using
   Our standard way, I suppose. dev cycle on staging, release to prod, try to do hotfixes on both
- [x]  does their email server work?
   can we even find out before the domain name switchover happens?
   yes, should do - will only find out when there's a domain name
- [x]  dataset previews - map, graph, etc. what do
- [x] what are we doing with domain names?
   data.dublinked.ie -> data.smartdublin.ie
   dublinked.ie -> data.smartdublin.ie
   data.smartdublin.ie
   wordpress just change IP
   still keep dublinked.ie email addresses working. it would be better if they moved from those to dublinked@smartdublin.ie though.
- [x] front page design
   categories similar to dgi
   similar to hse, focus on areas, wastewater etc
   featured datasets & use cases
   not necessarily a dataset viz
- [x] data quality actions
- [x] wordpress delta migration from current smartdublin to our one needs to be done again by spiral
- [x] what are we doing about users? are they able to login etc?
   ask smartdublin guys. it's probably good enough to just create new users for those that want them.
- [x] RTPI - either workaround or get them to fix, or both
- [x] migrate stories from dublinked wordpress to smartdublin

# Release day
- [ ] new relic, uptimerobot, etc
   already on the production server. needs to be brought back into the docker config though.
- [x] journey times script
   Basically just don't forget to email them when we go live :)

# Post-release
- [ ] editable front page segments
- [ ] feature/pinned datasets
